### ISSUE TITLE
[bug]: the "models.yaml" file is not created at the right path

### ISSUE THREAD

**Issue by metal3d** [invoke-ai/InvokeAI#2779]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### VRAM

15GB

### What happened?

Using Collab, to try to make textual inversion, I manually installed Invoke AI with these commands:

```
!pip install InvokeAI[xformers] --use-pep517 --extra-index-url https://download.pytorch.org/whl/cu117
!invokeai-configure --root_dir /content/invokeai --yes 
```

The root directory is set to `/content/invokeai`.

The problem is that the output says this:
```
Successfully created new configuration file /root/invokeai/configs/models.yaml
```

That means that the `models.yaml` file is not placed at the `root_dir`.

This breaks all the possible commands (image generation, textual inversion, ...) 

Of course, copying the file at the right place works. Textual inversion can start on Premium colab ;)

**Comment by metal3d** [comment]
FYI
![image](https://user-images.githubusercontent.com/100080/221048525-cbd36568-cb0a-4426-b32f-4830b5a40556.png)

**Comment by mickr777** [comment]
might be related too #2577

**Comment by lstein** [comment]
Is this with the released 2.3.0 or with unreleased 2.3.1? If the latter, which release candidate did you observe the problem on?

I am testing on my end to see if I can reproduce.

**Comment by lstein** [comment]
Also, are you using a virtual environment?

**Comment by lstein** [comment]
Sorry - asked the wrong questions. I see you were doing a network install. I'll try it out with the release candidate to see if the problem is fixed.

**Comment by lstein** [comment]
So the problem doesn't occur when I do a network install using 2.3.1-rc4 inside a docker container, but it may be a specific problem with collab. Could you check whether the problem persists when you use the release candidate? The install incantation is:
```
pip install InvokeAI[xformers]==2.3.1-rc4 --use-pep517 --extra-index-url https://download.pytorch.org/whl/cu117
```
Followed by `invokeai-configure` with the options you used earlier.

**Comment by metal3d** [comment]
Hello. Thanks for your quick answer.
I will try today.

PS : to automate the model installation via `import_model` command, is there a way to not open the invokeai terminal?

**Comment by metal3d** [comment]
There is an error, even with `--help`:

```
Loading Python libraries...

╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /usr/local/bin/invokeai-configure:5 in <module>                              │
│                                                                              │
│   2 # -*- coding: utf-8 -*-                                                  │
│   3 import re                                                                │
│   4 import sys                                                               │
│ ❱ 5 from ldm.invoke.config.invokeai_configure import main                    │
│   6 if __name__ == '__main__':                                               │
│   7 │   sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])     │
│   8 │   sys.exit(main())                                                     │
│                                                                              │
│ /usr/local/lib/python3.9/dist-packages/ldm/invoke/config/invokeai_configure. │
│ py:40 in <module>                                                            │
│                                                                              │
│    37                                                                        │
│    38 import invokeai.configs as configs                                     │
│    39                                                                        │
│ ❱  40 from ..args import PRECISION_CHOICES, Args                             │
│    41 from ..globals import Globals, global_config_dir, global_config_file,  │
│    42 from .model_install import addModelsForm, process_and_execute          │
│    43 from .model_install_backend import (                                   │
│                                                                              │
│ /usr/local/lib/python3.9/dist-packages/ldm/invoke/args.py:100 in <module>    │
│                                                                              │
│     97                                                                       │
│     98 import ldm.invoke                                                     │
│     99 import ldm.invoke.pngwriter                                           │
│ ❱  100 from ldm.invoke.conditioning import split_weighted_subprompts         │
│    101                                                                       │
│    102 from ldm.invoke.globals import Globals                                │
│    103                                                                       │
│                                                                              │
│ /usr/local/lib/python3.9/dist-packages/ldm/invoke/conditioning.py:91 in      │
│ <module>                                                                     │
│                                                                              │
│    88 │                                                                      │
│    89 │   return positive_prompt, negative_prompt                            │
│    90                                                                        │
│ ❱  91 def get_max_token_count(tokenizer, prompt: FlattenedPrompt|Blend, trun │
│    92 │   if type(prompt) is Blend:                                          │
│    93 │   │   blend: Blend = prompt                                          │
│    94 │   │   return max([get_max_token_count(tokenizer, c, truncate_if_too_ │
╰──────────────────────────────────────────────────────────────────────────────╯
TypeError: unsupported operand type(s) for |: 'type' and 'type'
```

**Comment by metal3d** [comment]
It's OK with Python 3.10 on colab.

**Comment by lstein** [comment]
We (think) we fixed the Python 3.9 bug before the release went out. Could
you give it a try again with Colab running Python 3.9?

Lincoln

**Comment by metal3d** [comment]
I can 😉
I will make some others tests tomorrow.

Thanks a lot for all.
